core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -415,7 +416,7 @@ class HttpGoogleServicesDAO(
           retryWhen500orGoogleError(() => {
             Option(executeGoogleRequest(fetcher).getItems)
           }) flatMap {
-            case None => Future.successful(BigInt(0))
+            case None => Future.successful(BucketUsageResponse(BigInt(0), DateTime.now()))
```

Comment:
🤔 I'm not grokking the flow here -- is this when there are no objects in the bucket?

Reply:
Yes exactly- at this point we haven't found a storage log for the bucket, so we do a quick check with `getStorage(getBucketServiceAccountCredential).objects.list(bucketName).setMaxResults(1L)` to verify that the workspace bucket is actually empty

Reply:
Interesting. So it sounds like this would return an error for the first ~24 hours of a workspace's existence (until the first usage log gets generated) if the workspace bucket contains any files? I wonder how we handle that in the UI. That's potentially not a good experience for a new user.

Reply:
Shockingly it doesn't result in an error being bubbled up. I'll do more digging, there's something else at play here it seems

Reply:
Maybe a storage log gets generated when a bucket is created?

Reply:
Spot on.

https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala#L246

Called [here](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala#L286) during workspace setup.